system/lvol.py

```diff
@@ -353,6 +367,9 @@ def main():
                     else:
                         module.fail_json(msg="Unable to resize %s to %s%s" % (lv, size, size_unit), rc=rc, err=err)
 
+        elif not size:
```

Comment:
The if/elif/else order here looks a bit fragile, as it depends on default `size_opt = 'L'` on line 209. Maybe we should check if size if specified before `size_opt == 'l'` check on line 330?